gui/src/app/menu.rs

```diff
@@ -10,4 +10,5 @@ pub enum Menu {
     CreateSpendTx,
     Recovery,
     RefreshCoins(Vec<OutPoint>),
+    PsbtPreSelected(Txid),
```

Comment:
Maybe one day, we would separate list panels from list item modal, like a Menu::PSBTs and Menu::PSBT(txid) and keep all the panels states in memory instead of having one app current state.